notification/README.md

```diff
@@ -0,0 +1,253 @@
+--- 
+layout: pattern 
+title: Notification
+category: Behavioural
+language: en 
+tags:
+- Decoupling
+- Presentation
+- Domain
+---
```

Comment:
The frontmatter needs still adjustments. If you look at the requirements at https://github.com/iluwatar/java-design-patterns/wiki/01.-How-to-contribute, you'll notice that there is no `layout` and `tags` should be renamed to `tag`.